docs/installation.md

```diff
@@ -1,8 +1,15 @@
 # Installation
 
-Install React Table as a dependency using `npm` or `yarn`
+Install React Table as a dependency using `npm` or `yarn`. If you'd like to
+support the maintainers of React Table, please consider enabling installation
+analytics with [Scarf](https://www.npmjs.com/package/@scarf/scarf) by setting
+the environment variable `SCARF_ANALYTICS=true` before you install. Analytics
+are turned off by default.
 
 ```bash
+# If you'd like to support react-table by sending installation analytics:
+$ export SCARF_ANALYTICS=true
+
```

Comment:
I like that the analytics are disabled by default for now, but I wish the mechanism for opting in was easier or more fluid. I was thinking something like a post-install hook that would just prompt the user for their permissions to collect analytics like this:

```bash
$ Would you like to send usage/support analytics to React Table? (Y/n): _
```

They hit enter for yes (or change it), and it edits the package.json file to denote their choice. Thoughts?

Reply:
Great idea! We can get this added to scarf-js. We'll just need to make sure that wait time for user input isn't too long to avoid slowing down any builds, but a short wait for input seems fine.

Reply:
Is this trivial? I would love to wait for this if possible.

Reply:
Totally possible! It's not quite trivial, but definitely straight-forward. Working on this now and will have it ready soon.

Reply:
The PR is just about ready to go, just had one question for you in https://github.com/scarf-sh/scarf-js/issues/4#issuecomment-600922194